target/linux/ath79/nand/base-files/lib/upgrade/platform.sh

```diff
@@ -1,24 +1,93 @@
+#!/bin/sh
+
+# SPDX-License-Identifier: GPL-2.0
 #
-# Copyright (C) 2011 OpenWrt.org
+# Copyright (C) 2019 Jeff Kletsky
```

Comment:
License change and copyright change. You will have to ask everybody ...

Reply:
No it is a full replace with code of my authorship. If anything, _that_ code is a violation as my code has been around for months now.

Reply:
Well, what's been there before your changes is just a copy from common base-files, which most probably has been copied several times before that. So, the only thing taken from your inspiration before the changes done here is the use of split base-files (testified by wrongly copied k2t.sh).
Considering the topic replace vs. addition of copyright I would have to do research on what's legitimate in this case.

Reply:
The point is that it is a full replace, with the exception of a couple of lines that fall under the merger doctrine. _There is only one or very few ways to express an idea._